src/lib/controller/message_handler.ts

```diff
@@ -298,28 +296,29 @@ export class ControllerMessageHandler {
         };
       }
       case ControllerCommand.beginOTAFirmwareUpdate: {
-        success = await driver.controller.firmwareUpdateOTA(message.nodeId, [
-          message.update,
-        ]);
-        return { success };
+        const result = await driver.controller.firmwareUpdateOTA(
+          message.nodeId,
+          [message.update]
+        );
+        return { result };
       }
```

Comment:
This command is weird, since it does something different than the name suggests. I assume it is left in because of backwards compatibility?